src/Workspaces/CSharpTest/CSharpServicesTest.csproj

```diff
@@ -30,6 +30,8 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Web.Extensions" />
```

Comment:
intentional.  each of these has a json parser that we reference in order to validate our own parser against.